package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "postcss-css-variables",
-  "version": "0.12.0",
+  "version": "0.12.1",
```

Comment:
Remove this version bump in the pull request. We will bump it when we push out a release via `npm version` command